spec/models/test_track/analytics/safe_wrapper_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe TestTrack::Analytics::SafeWrapper do
-  let(:underlying) { double("Client", track_assignment: true, sign_up!: true) }
+  let(:underlying) { double("Client", track: true, sign_up!: true) }
 
   subject { TestTrack::Analytics::SafeWrapper.new(underlying) }
 
-  describe '#track_assignment' do
+  describe '#track' do
     it 'calls underlying' do
-      expect(subject.track_assignment(123, foo: "bar")).to eq true
-      expect(underlying).to have_received(:track_assignment).with(123, foo: "bar")
+      expect(subject.track("my event")).to eq true
```

Comment:
We don’t care that this is no longer a true representation of the contract do we? I think it’s fine.

Reply:
do you mean you'd prefer I use a mock that looks more like an event?